client/src/components/user-profile.jsx

```diff
@@ -12,10 +12,10 @@ function Proflie(props) {
     // console.log(props.user.interests);
     return (
         <div>
-            <img src={props.user.profilePicURL} />
-            <h1>@{props.user.username}</h1>
-            <h2>{props.user.name}</h2>
-            <ul>{props.user.interests.map(interest => <li>{interest}</li>)}</ul>
+            <img class="img-thumbnail" src={props.user.profilePicURL} />
+            <h2><strong><small>@ </small>{props.user.username}</strong></h2>
```

Comment:
all styling should be done in a separate css file, either from bootstrap theme or our own custom css in a style.css file that we haven't created yet